src/CHANGES.txt

```diff
@@ -66,6 +66,9 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       ParseFlags: -iquote and -idirafter.
     - Fix more re patterns that contain \ but not specified as raw strings
       (affects scanners for D, LaTeX, swig)
+      
+    From Michael Hartmann:
+    - Fix handling of Visual Studio cross compilers: correctly handle empty host_trgt_dir
```

Comment:
Actually (if I'm reading this correctly), this handles when the specified (host_platform,target_platform) tuple isn't a known one by correctly exiting?

If so a unit test should be fairly simple to create.